src/textual/app.py

```diff
@@ -1247,14 +1248,27 @@ async def _on_css_change(self) -> None:
                 stylesheet.read_all(css_paths)
                 stylesheet.parse()
                 elapsed = (perf_counter() - time) * 1000
+                if self._css_has_errors:
+                    from rich.panel import Panel
+
+                    self.log.system(
+                        Panel(
+                            "CSS files successfully loaded after previous error:\n\n- "
+                            + "\n- ".join(str(path) for path in css_paths),
+                            style="green",
+                            border_style="green",
+                        )
+                    )
```

Comment:
Shouldn't this be in the `else` statement already?

Reply:
Already? Not sure I follow. This is something new. Extra bit of logging to go with the existing logging.

Reply:
Ignore the "already", then.
Shouldn't this be the first piece of code in the `else` instead of the last piece of code in the `try`?

Reply:
Ahh I see. That would work too, but I was keeping the logging with the existing logging.

Reply:
Maybe both logs should be in the `else` statement. 🤷

Reply:
That would work too, of course, but I feel keeping the reporting of the elapsed time closer to the calculation of it makes for an easier read.